src/components/Footer/Footer.module.css

```diff
@@ -1,12 +1,12 @@
-.footer {
```

Comment:
Why are we removing these spaces? That doesn't align with our code formatting configuration.